migration/2.1.3/SequencerInbox.sol

```diff
@@ -0,0 +1,843 @@
+// Copyright 2021-2022, Offchain Labs, Inc.
```

Comment:
Where did we get this contract from? Is this like copied from v2.1.3 repo?

Reply:
I believe I copied the other revert contract, the one now stored in the `2.1.0` folder, and added the changes needed to make this compatible with the vanilla version of the 2.1.3 sequencer inbox.